app/views/games/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<br>
+<br>
+<%= content_tag :div, class: "piece_pos", data: { positions: @piece_positions } do %>
+<% end %>
+<div class="text-center players">
+  <h2>&#9819; Player1 vs &#9813; Player2</h2>
```

Comment:
Should add the dynamic player names now. I don't see a ticket for it, and it's as simple as a few content_tags as above